sh_scrapy/crawl.py

```diff
@@ -155,6 +155,29 @@ def _launch():
     _run_usercode(job['spider'], args, _get_apisettings, loghdlr)
 
 
+def list_spiders():
+    """ An entrypoint for list-spiders."""
+    try:
+        from scrapy.exceptions import ScrapyDeprecationWarning
+        warnings.filterwarnings(
+            'ignore', category=ScrapyDeprecationWarning, module='^sh_scrapy')
+        from sh_scrapy.env import get_scrapy_list_args_and_env, decode_uri
+        job_settings = decode_uri(envvar='JOB_SETTINGS')
```

Comment:
@dangra That was the main reason why I passed project in `JOB_DATA` before, to avoid double decoding `JOB_SETTINGS` env.